spec/models/conference_spec.rb

```diff
@@ -3,10 +3,15 @@
 RSpec.describe Conference do
   it_behaves_like 'HasSeason'
 
-  it { is_expected.to have_many(:attendances) }
-  it { is_expected.to have_many(:attendees) }
-  it { is_expected.to validate_presence_of(:name) }
-  it { is_expected.to validate_presence_of(:round) }
+  describe 'dependencies' do
+    subject { FactoryGirl.build(:conference) }
+
+    it { is_expected.to have_many(:attendances) }
+    it { is_expected.to have_many(:attendees) }
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:round) }
+    # The custom date validator is tested in test 'chronology', below
+  end
```

Comment:
Why did it fail? I have an idea: the `data_interval` does a `starts_on > ends_on`. It will fail if either is `nil`.  See comment below.